test/e2e/testnet/node.go

```diff
@@ -268,7 +267,19 @@ func (n *Node) Start() error {
 		return err
 	}
 
-	return n.forwardPorts()
+	err, rpcProxyHost := n.Instance.AddHost(rpcPort)
+	if err != nil {
+		return err
+	}
+	n.rpcProxyHost = rpcProxyHost
+
+	err, grpcProxyHost := n.Instance.AddHost(grpcPort)
+	if err != nil {
+		return err
+	}
+	n.grpcProxyHost = grpcProxyHost
```

Comment:
This returns a host to reach the RPC/gRPC endpoint via the reverse proxy without relying on port-forwards.